auditbeat/helper/hasher/hasher_test.go

```diff
@@ -28,14 +27,14 @@ import (
 )
 
 func TestHasher(t *testing.T) {
-	dir, err := ioutil.TempDir("", "auditbeat-hasher-test")
+	dir, err := os.MkdirTemp("", "auditbeat-hasher-test")
```

Comment:
This and the deferred `os.Removell` can be replaced with https://pkg.go.dev/testing#T.TempDir.